src/classes/TDTM_TriggerScaffolds_TEST.cls

```diff
@@ -43,14 +43,62 @@ public with sharing class TDTM_TriggerScaffolds_TEST {
     	//just for test coverage, and to verify no exception gets thrown.
     	insert new Lead(Company = 'Pencil Makers', LastName = 'Smith');
     	
-    	System.assert(true); //Just checking that we get to this point = no exception thrown
+    	System.assert(true, 'No exception thrown'); //Just checking that we get to this point = no exception thrown
     }
     
     public static testmethod void tdtmCampaign() {
     	//Currently we don't have any Trigger_Handler__c record for Campaign. This test is 
```

Comment:
@ceiroa very minor, i think the indentation here looks weird...

Reply:
I think I have fixed the indentation, @xinnanlisf. Let me know what you think. Thanks!